server/jetstream_cluster.go

```diff
@@ -1108,7 +1120,7 @@ func (js *jetStream) createRaftGroup(rg *raftGroup) error {
 
 	storeDir := path.Join(js.config.StoreDir, sysAcc.Name, defaultStoreDirName, rg.Name)
 	fs, err := newFileStore(
-		FileStoreConfig{StoreDir: storeDir, BlockSize: 8_000_000, AsyncFlush: true},
+		FileStoreConfig{StoreDir: storeDir, BlockSize: 4_000_000, AsyncFlush: true, SyncInterval: 2 * time.Minute},
```

Comment:
Maybe we should expose through config the sync interval so it can be disabled if performance impact is too much. One could rely on the idea that with R<n> it would need several failures to lose data.

Reply:
These are for the internal WAL, so probably would not expose them.